dms/models/dms_file.py

```diff
@@ -411,10 +411,10 @@ def _compute_extension(self):
     @api.depends("name", "content")
     def _compute_mimetype(self):
         for record in self:
-            binary = base64.b64decode(record.with_context({}).content or "")
-            record.res_mimetype = guess_mimetype(
-                binary, default="application/octet-stream"
-            )
+            if record.content:
+                record.res_mimetype = guess_mimetype(
+                    record.content, default="application/octet-stream"
+                )
```

Comment:
and the else part?

Reply:
It's really necessary? It's a compute and content field is required.

Reply:
I think so, unless a default value is provided in the field definition. If not, you will be hit by `Missing cache value` sooner or later.

Reply:
Ok, change it. 
Remove default because is the same that in function https://github.com/odoo/odoo/blob/13.0/odoo/tools/mimetypes.py#L136